watchtower-plugin/src/dbm.rs

```diff
@@ -316,7 +325,32 @@ impl DBM {
         tx.commit()
     }
 
-    /// Loads the appointment receipts associated to a given tower
+    /// Loads a given appointment receipt of a given tower from the database.
+    pub fn load_appointment_receipt(
+        &self,
+        tower_id: TowerId,
+        locator: Locator,
+    ) -> Result<AppointmentReceipt, Error> {
+        let mut stmt = self
+            .connection
+            .prepare("SELECT * FROM appointment_receipts WHERE tower_id = ?1 and locator = ?2")
```

Comment:
I would suggest to explicitly state de columns in the SELECT session here.
This might avoid future errore where db columns are reshuffled and, for instance, ``start_block`` is no longer the first column of the db

Reply:
Rebased to fix this. Will also open an issue to address the rest of the `SELECT *` matches in the codebase.

Reply:
After the rebase this specific hit was not fixed, given it was already merged in #83. I've fixed the only one introduced by this PR and will leave the rest to be fixed by the aforementioned issue.